### ISSUE TITLE
Disable logging?

### ISSUE THREAD

**Issue by Thell** [sedwards2009/extraterm#267]
When starting from command line I see
![image](https://user-images.githubusercontent.com/2185020/83672810-7d117a00-a59c-11ea-910e-34ff82943366.png)
and redirecting `2>/dev/null` it still prints to screen and I don't see anything in `--help` or app settings about log level and a search on the site and in issues doesn't come up with anything.

Is this normal? If not I'll provide more detail.
Can it be disabled? If so, how?

Thank you.

**Comment by sedwards2009** [comment]
Yes this is normal, and no, there is no easy way to stop it. The general idea is that you wouldn't be starting your terminal from another terminal. Unless of course you're an Extraterm developer in which case the logging is useful.

**Comment by Thell** [comment]
Thanks for the explanation and completely understood.

The use case here was that I am setting up wsl2 ubuntu and using RStudio which has the capability to open a terminal from the editor but unfortunately getting _that_ to open Windows Terminal isn't really an option as of yet, and Terminator looks so old school coming from Windows Terminal so extraterm looked like a perfect fit but those messages print out to the main Windows Terminal and/or the RStudio built-in terminal which calls the system terminal and without a way to quiet it down it is just noise.
If you ever get a the whim to make logging an option... ;)

**Comment by sedwards2009** [comment]
@Thell  thanks for the context. Does redirecting normal output to `/dev/null` work? Adding some kind of `--quiet` flag wouldn't be too hard.

**Comment by Thell** [comment]
`Info` looks to be `stdout` and `warn` -> `stderr` so redirecting both `> /dev/null 2>&1` works when opening from Windows Terminal but not when starting from the RStudio interface. Like you said, its not really the use case it was designed for. It is an interesting choice to have verbosity hard-coded. :)

On the application side (RStudio) I guess it has to do with properly passing the argument from the param field (which, without looking, I'd guess gets parsed to a string and munged before the system sees it).
